tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -55,14 +55,13 @@ public function testFix($expected, $input = null)
     /**
      * @param string      $expected
      * @param null|string $input
-     * @param null|string $extraSpaceInput
```

Comment:
`$extraSpaceInput` does not exist as param, so I removed the hint.
Line below I changed `mixed` to `string` as this is more strict.